src/chainparams.cpp

```diff
@@ -92,10 +92,10 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT; 
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000000000000000000200020");
+        consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000000000000000136e15425");
```

Comment:
A bit high given there are no specialised GPU/FPGA miners available in addition to small number of miners.

Reply:
> A bit high given there are no specialised GPU/FPGA miners available in addition to small number of miners.

No. That is the total amount of work needed to produce the chain
0x0000000000000000000000000000000000000000000000000000000136e15425 
is 5215704101 hashes, which given an estimated 3 days of work is an average of 20kH/s. The current hashrate is 3MH/s likely from a midrange GPU or CPU cluster. which can clear this value in just 30 minutes.

Reply:
> > A bit high given there are no specialised GPU/FPGA miners available in addition to small number of miners.
> 
> No. That is the total amount of work needed to produce the chain 0x0000000000000000000000000000000000000000000000000000000136e15425 is 5215704101 hashes, which given an estimated 3 days of work is an average of 20kH/s. The current hashrate is 3MH/s likely from a midrange GPU or CPU cluster. which can clear this value in just 30 minutes.

Hello, `0000000000000000000000000000000000000000000000000000000136e15425`  is the chainwork from block height 4072 - used for the last checkpoint.